remoteFortisoar/info.json

```diff
@@ -0,0 +1,145 @@
+                            "type": "text",
+                            "name": "password",
+                            "value": ""
+                        }
+                    ],
+                    "HMAC": [
+                        {
+                            "description": "Private key used to authenticate FortiSOAR Server",
+                            "title": "Private Key",
+                            "tooltip": "Private key used authenticate FortiSOAR Server",
+                            "name": "private_key",
+                            "type": "textarea",
+                            "visible": true,
+                            "editable": true,
+                            "required": true,
+                            "value": ""
+                        },
+                        {
+                            "description": "Public key used to authenticate FortiSOAR Server",
+                            "title": "Public Key",
+                            "tooltip": "Public key used to authenticate FortiSOAR Server",
+                            "required": true,
+                            "editable": true,
+                            "visible": false,
+                            "type": "textarea",
+                            "name": "public_key",
+                            "value": ""
+                        }
+                    ]
+                }
+            },
+            {
+                "title": "Verify SSL",
+                "type": "checkbox",
+                "name": "verify_ssl",
+                "required": false,
+                "visible": true,
+                "editable": true
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "make_api_call",
+            "title": "Make an API call",
+            "description": "Makes an API call to the given endpoint with the given method.",
+            "category": "",
+            "annotation": "",
+            "output_schema": {
+                "result": "",
+                "api_data": ""
+            },
+            "enabled": true,
+            "parameters": [
+                {
+                    "title": "HTTP method",
+                    "name": "method",
+                    "description": "Http method",
+                    "type": "select",
+                    "visible": true,
+                    "editable": true,
+                    "required": true,
+                    "options": [
+                        "GET",
+                        "POST",
+                        "PUT",
+                        "DELETE"
+                    ]
+                },
+                {
+                    "title": "Endpoint IRI",
+                    "type": "text",
+                    "name": "iri",
+                    "required": true,
+                    "visible": true,
+                    "editable": true
+                },
+                {
+                    "title": "Data",
```

Comment:
We should provision for request payload, query parameters, and file input.

Reply:
Created mantis #0883479 to track this requirement